deepchem/rl/a3c.py

```diff
@@ -21,10 +21,11 @@ def __init__(self, value_weight, entropy_weight, **kwargs):
     self.entropy_weight = entropy_weight
 
   def create_tensor(self, **kwargs):
-    reward, action, prob, value = [layer.out_tensor for layer in self.in_layers]
-    log_prob = tf.log(prob)
-    policy_loss = -tf.reduce_sum(
-        (reward - value) * tf.reduce_sum(action * log_prob))
+    reward, action, prob, value, advantages = [
+        layer.out_tensor for layer in self.in_layers
+    ]
+    log_prob = tf.log(prob + 0.0001)
+    policy_loss = -tf.reduce_sum(advantages * tf.reduce_sum(action * log_prob))
```

Comment:
@peastman you were right there was a value_weight sweat spot between 4-6.

![selection_003](https://cloud.githubusercontent.com/assets/1225605/26462873/8689eecc-4150-11e7-898f-359912b3788b.png)

![selection_004](https://cloud.githubusercontent.com/assets/1225605/26462877/8af1435c-4150-11e7-9996-dba9b891fe08.png)

The model isn't as stable as with lambda=0 but if you use a pocket algorithm it can produce similar results.